Account/FxAClient10.swift

```diff
@@ -181,7 +181,7 @@ open class FxAClient10 {
         }
 
         let ciphertext = data.subdata(in: Range(uncheckedBounds: (lower: 0 * KeyLength, upper:  2 * KeyLength)))
-        let MAC = data.subdata(in: Range(uncheckedBounds: (lower: 2 * KeyLength, upper: 1 * KeyLength)))
+        let MAC = data.subdata(in: Range(uncheckedBounds: (lower: 1 * KeyLength, upper: 2 * KeyLength)))
```

Comment:
I wonder how this worked at all? Seems really weird to have the lower be higher than the upper in the first place...